packages/paste-core/components/checkbox/__tests__/checkbox.test.tsx

```diff
@@ -34,8 +34,11 @@ const defaultGroupProps = {
 
 describe('Checkbox', () => {
   it('should render', () => {
-    const {getByRole} = render(<Checkbox {...defaultProps}>foo</Checkbox>);
+    const {getByRole, container} = render(<Checkbox {...defaultProps}>foo</Checkbox>);
     expect(getByRole('checkbox')).not.toBeNull();
+
+    const checkIcon = container.querySelector('[data-paste-element="CHECKBOX_ICON"');
```

Comment:
I guess it doesn't matter bc the tests are passing, but aren't these queries missing the closing bracket?
`[data-paste-element="CHECKBOX_ICON"'`
`[data-paste-element="CHECKBOX_ICON"]'`

Reply:
Ah thanks, great catch!